tox.ini

```diff
@@ -1,34 +1,32 @@
 [tox]
 skipsdist = True
-envlist =
-    py{27,33,34,35}-{test,lint},
-    # pylint is broken under py36, so skip linting for now.
-    py36-test
+envlist = py{27,33,34,35,36}
```

Comment:
I've combined the test and lint commands because the versions of Python 2.7 and Python 3.3 on Travis did not like my egg writers being in their own virtualenv. This was the easiest way to get around the issue.